react-app/src/components/organisms/MovieList/MovieListComponent.tsx

```diff
@@ -0,0 +1,22 @@
+import React, { FunctionComponent } from 'react';
+
+import { IMovie } from '../../molecules/MovieCard/IMovie';
+import MovieCardComponent from '../../molecules/MovieCard/MovieCardComponent';
+import { IMovieList } from './IMovieList';
+import ErrorBoundary from '../../atoms/ErrorBoundary/ErrorBoundary';
+import MoviesWrapper from './MovieList';
+
+const MovieListComponent: FunctionComponent<IMovieList> = ({ items }) => (
+  <ErrorBoundary>
```

Comment:
I would recommend to wrap with `<ErrorBoundary />` the wider part of your app to catch errors from ***any*** source, since keeping UI controls (like search bar and add movie button) doesn't make much sense if the essential part of your app (movie list) crashes